material/base.html

```diff
@@ -86,8 +86,19 @@
       {% include "partials/header.html" %}
     {% endblock %}
     <div class="md-container">
-      {% set feature = config.extra.get("feature", {}) %}
-      {% if feature.tabs %}
+      <div class="md-banner"> 
+        <div class="md-banner__inner md-grid">
```

Comment:
What do you think about abstracting this to a partial? I'd really like to customise this section of the site and with the banner feature I can see a lot of additional opportunities!